src/System.Private.CoreLib/src/System/Threading/ThreadPool.cs

```diff
@@ -652,17 +652,6 @@ internal static bool Dispatch()
             }
             catch (ThreadAbortException tae)
```

Comment:
This catch block should be removed altogether - it can be done in follow up change.

Reply:
Will leave that to @filipnavara who is working on TP

Reply:
Already done in my last iteration of the ThreadPool PR.